epub-a11y-meta-guide/1.0/draft/index.html

```diff
@@ -180,3106 +254,2739 @@ <h3>General application guidelines</h3>
+						is declared in the package document.</p>
+
+					<aside class="example" title="Package document markup for a auditory access mode">
+						<pre>&lt;package xmlns="http://www.idpf.org/2007/opf" xml:lang="en">
+   &lt;metadata>
+      &#8230;
+      &lt;meta property="schema:accessMode">auditory&lt;/meta>
+      &#8230;
+   &lt;/metadata>
+   &#8230;
+&lt;/package></pre>
+					</aside>
 
-							<p>And finally, since there is a content image present, you would also include:</p>
+					<p>Not all auditory content in a publication carries information necessary to its understanding.
+						Examples of when an auditory access mode would not be set include:</p>
+
+					<ul>
+						<li>for mood music that plays in the background while reading;</li>
+						<li>for background music in an instructional video when all the information is conveyed
+							visually;</li>
+						<li>for the background sound to an interactive game when it is solely for entertainment
+							value;</li>
+					</ul>
+
+					<section id="accessMode-sync">
+						<h6>Synchronized text and audio</h6>
+
+						<p>A special case for audio in EPUB 3 involves text and audio synchronization using the <a
+								href="https://www.w3.org/TR/epub/#sec-media-overlays">media overlays feature</a>
+							[[epub-3]]. Setting the correct access modes and sufficient access modes for EPUB 3
+							publications that contain synchronized text-audio playback requires evaluating whether
+							playback is essential to reading the publication or an additional feature.</p>
+
+						<p>One of the more common features of media overlays, particularly in mainstream publishing, is
+							to provide synchronized playback of the full text of a publication with full audio
+							narration. These types of publications are commonly referred to as "read aloud" books, as
+							the user chooses whether or not to turn on the narration (unlike traditional audiobooks
+							where only the audio is available).</p>
+
+						<p>In this case, because the audio playback is an extra feature, and is not required for reading
+							the publication, it is <strong>not</strong> listed as an auditory access mode. Instead, the
+							presence of text and audio synchronization is declared using the <a
+								href="#synchronizedAudioText"><code>synchronizedAudioText</code> accessibility
+								feature</a>.</p>
+
+						<p>Having full audio playback capability in this case instead mean that an <a
+								href="#ams-auditory">auditory sufficient access mode</a> needs to be declared — the user
+							can listen to the complete publication as an alternative to reading the text.</p>
 
-							<pre>&lt;meta property="schema:accessMode"&gt;visual&lt;/meta&gt;</pre>
+						<aside class="example"
+							title="Package document markup for a book with full text and audio synchronization">
+							<pre>&lt;meta property="schema:accessMode">textual&lt;/meta>
```

Comment:
Don't we also need accessMode of "auditory" as well here? I don't think we can include an accessModeSufficient without it also being one of the AccessModes right?

Reply:
There's data in ace that can be used to identify access modes. That's what I use in smart to disable success criteria that don't apply to a publication and set some of the metadata. It's not perfect, though, as you say, because it might tell you there are videos, for example, but it can't tell you if they have audio, or how essential any of the content is to reading the publication.